frontend/constants.js

```diff
@@ -3,8 +3,11 @@ import getConfig from './helpers/getConfig';
 const {
   headerHomePageBackgroundColor,
   headerHomePageTextColor,
+  backgroundColor,
+  textColor,
 } = getConfig();
 
-export const HEADER_BACKGROUND_COLOR = headerHomePageBackgroundColor || null;
-export const HEADER_COLOR = headerHomePageTextColor || null;
+// Use old config values headerHomePageBackgroundColor, headerHomePageTextColor if they still exist
+export const HEADER_BACKGROUND_COLOR = headerHomePageBackgroundColor || backgroundColor || null;
```

Comment:
If we provide backwards compatibility then it still should be 2.0.0 and extension config should be adjusted to still keep the old configs.

If not, please remove it since it's introducing TD we don't need.

Reply:
removed